crates/sui-graphql-rpc/src/main.rs

```diff
@@ -14,20 +14,8 @@ use sui_graphql_rpc::server::graphiql_server::start_graphiql_server;
 use tokio_util::sync::CancellationToken;
 use tokio_util::task::TaskTracker;
 
-// WARNING!!!
-//
-// Do not move or use similar logic to generate git revision information outside of a binary entry
-// point (e.g. main.rs). Placing the below logic into a library can result in unnecessary builds.
-const GIT_REVISION: &str = {
-    if let Some(revision) = option_env!("GIT_REVISION") {
-        revision
-    } else {
-        git_version::git_version!(
-            args = ["--always", "--abbrev=40", "--dirty", "--exclude", "*"],
-            fallback = "DIRTY"
-        )
-    }
-};
+// Define the `GIT_REVISION` const
+bin_version::git_revision!();
```

Comment:
Please note that for GraphQL we have set the `abbrev` to 40 as per @ebmifa's request. Can we pass an argument to this `git_revision` to specify the length of the `sha`?

Reply:
What's the rationale for this vs having it be consistent with what we do elsewhere?

Reply:
Makes it simpler for deployments. @ebmifa could probably chime in more.

Reply:
I don't remember why I wanted a full sha now 🤦🏼 
But, for consistency's sake, let's keep it abbreviated for now.

Reply:
Ok, then I think somewhere in the GraphQL deployments/CI stuff we need to get the full SHA for the rest to work. You might remember where, so just mentioning this.